AdobeStockAdminUi/Block/Adminhtml/System/Config/TestConnection.php

```diff
@@ -8,17 +8,19 @@
 
 namespace Magento\AdobeStockAdminUi\Block\Adminhtml\System\Config;
 
+use Magento\Backend\Block\Template\Context;
 use Magento\AdobeImsApi\Api\ConfigInterface;
-use Magento\AdobeStockClientApi\Api\ClientInterface;
 use Magento\Config\Block\System\Config\Form\Field;
-use Magento\Backend\Block\Template\Context;
+use Magento\AdobeStockClientApi\Api\ClientInterface;
```

Comment:
I think it's better to avoid changing the order of `use` statements without a reason.